app/src/main/java/ch/epfl/sdp/game/Game.java

```diff
@@ -38,7 +39,6 @@ private Game() {
         updatables = new ArrayList<>();
         displayables = new ArrayList<>();
         scoreUpdater = new ScoreUpdater();
-        displayablesOnce = new ArrayList<>();
```

Comment:
From what I see, there is no more notion of displayed once. So now all item are permanently displayed like the other displayables?

Reply:
The reason we want to display itemBox once is that the itemBox will never change it's location and we don't need to refresh it each game frame. Now I add a field "isDisplayed" to make sure the itemBox will be displayed once and not refresh it, when the itembox been taken by the player we remove it from maps

Reply:
I get the idea. Fine by me.